bftools-libs/meta.yaml

```diff
@@ -1,13 +1,15 @@
+{% set version = "6.6.0" %}
+
 package:
   name: "bftools-libs"
-  version: "6.6.0"
+  version: "{{ version }}{{ environ.get('VERSION_SUFFIX', '') }}"
```

Comment:
What's the use case for `{{ environ.get('VERSION_SUFFIX', '') }}"` and how would one set it up?

Reply:
I think that was used for testing purpose. Some repositories have it other don't.
Happy to keep it as simple as possible and use``{{ version }}`` only

Reply:
I don't know how to use it (esp. with the GitHub actions workflow) so I'll defer to whoever introduced it and knows more about it.

For comparison, in our recent experience with `bioformats2raw`, putting a suffix in the Jinja2 verison number e.g. https://github.com/ome/conda-bioformats2raw/blob/0.3.0rc3-0/bioformats2raw/meta.yaml was sufficient to get a pre-release deployed with the correct version although we introduced extra logic to use a the non-default label.

Reply:
IIRC it was to allow test packages to be pushed under a separate label on conda (e.g. have all main commits built and pushed to conda)- you can't have two packages with the same name in different labels. If you're not doing this you can get rid of it!

Reply:
Thanks @manics.  I would think it's safe to ignore for now.

@jburel you only modified `bftools/meta.yaml`. Can you modify this recipe as well?